mmv1/products/firebase/terraform.yaml

```diff
@@ -57,6 +57,26 @@ overrides: !ruby/object:Overrides::ResourceOverrides
           - project
     custom_code: !ruby/object:Provider::Terraform::CustomCode
       custom_import: templates/terraform/custom_import/self_link_as_name.erb
+  AppleApp: !ruby/object:Overrides::Terraform::ResourceOverride
+    import_format: ['{{project}} {{name}}']
```

Comment:
I think this should be:
```
'projects/{{project}}/iosApps/{{appId}}', 'iosApps/{{appId}}', '{{appId}}'] 
```

Since the resource's AppId is a GUID, we can support all 3 variants without ambiguity.